microsetta_interface/templates/new_results_page.jinja2

```diff
@@ -15,7 +15,9 @@
 <style>
     .nav-white-bg {
       background-color: #fff;
-      width: auto;
+      width: 100%;
+      padding-left: 100px;
+      padding-right: 100px;
```

Comment:
Container widths should be managed with the `width1 and `max-width` parameters, not `padding` or `margin`. This approach yields poor results on smaller screens.

Reply:
I originally used width but was met with unexpected behavior. I think it has to do with the stylings of the other divs, I'll see about arranging them differently for different results.